docs/guide/examples.rst

```diff
@@ -307,6 +309,36 @@ If your callback returns False, training is aborted early.
   plot_results([log_dir], timesteps, results_plotter.X_TIMESTEPS, "TD3 LunarLander")
   plt.show()
 
+Evaluating your agents
+^^^^^^^^^^^^^^^^^^^^^^
+To periodically evaluate an agent's performance on a separate test environment, use EvalCallback.
+You can control the evaluation frequency with eval_freq to monitor your agent's progress during training.
+
+.. code-block:: python
+
+  import os
+  import gymnasium as gym
+
+  from stable_baselines3 import SAC
+  from stable_baselines3.common.callbacks import EvalCallback
+
+  # Create log dir
+  eval_log_dir = "./eval_logs/"
+  os.makedirs(eval_log_dir, exist_ok=True)
+
+  # Initialize training environment with default parameters
+  train_env = gym.make("Pendulum-v1")
+
+  # Separate evaluation env, with different parameters
+  eval_env = gym.make("Pendulum-v1", g=7.0)
+
+  # Create callback that evaluates agent every 500 environment steps
```

Comment:
maybe add a note about dividing by the number of training env (see `EvalCallback` doc)